### PR TITLE
Landing page in EWS post sign in should be in custom report

### PR DESCRIPTION
**Pull request by kkrampa**
http://manage.dimagi.com/default.asp?186980#1044838

@czue Please read commit by commit. I can't find better solution for this. I'm open to any suggestions.

### CODE REVIEW

**Comment by kkrampa** [comment]
@czue Updated

File: settings.py [hunk 1]
```diff
@@ -1586,3 +1586,8 @@
     pass
 else:
     initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
+
+
+CUSTOM_DASHBOARD_PAGE_URL_NAMES = {
+    'ews-ghana': 'dashboard_page'
+}
```
Comment (czue) [inline settings.py]: can you move this closer to the other custom settings? e.g. maybe near `CUSTOM_MODULES`

**Comment by kkrampa** [comment]
@czue Done

**Comment by czue** [comment]
thx

**Pull request merged**